kotlin/usecases/serverless_rds/Readme.md

```diff
@@ -723,63 +501,91 @@ The following Kotlin code represents the **RetrieveItems** class that retrieves
         }
         return null
     }
+}
 
-    // Convert Work data into XML to pass back to the view.
-    private fun toXmlItem(id2: String, desc2: String, status2: String): Document? {
-        try {
-            val factory = DocumentBuilderFactory.newInstance()
-            val builder = factory.newDocumentBuilder()
-            val doc = builder.newDocument()
 
-            // Start building the XML.
-            val root = doc.createElement("Items")
-            doc.appendChild(root)
-            val item = doc.createElement("Item")
-            root.appendChild(item)
-
-            // Set Id.
-            val id = doc.createElement("Id")
-            id.appendChild(doc.createTextNode(id2))
-            item.appendChild(id)
-
-            // Set Description.
-            val desc = doc.createElement("Description")
-            desc.appendChild(doc.createTextNode(desc2))
-            item.appendChild(desc)
-
-            // Set Status.
-            val status = doc.createElement("Status")
-            status.appendChild(doc.createTextNode(status2))
-            item.appendChild(status)
-            return doc
-        } catch (e: ParserConfigurationException) {
-            e.printStackTrace()
+```
+
+### InjectWorkService class
+
+The following Kotlin code represents the **InjectWorkService** class. Notice that you need to specify ARN values for the secret manager and the Amazon Serverless Aurora database (as discussed in the *Creating the resources* section). Without both of these values, your code does not work. To use the **DSDataClient**, you need to create an **ExecuteStatementRequest** object and specify both ARN values, the database name, and the SQL statement used to submit data to the work table.
```

Comment:
the secret manager --> Secrets Manager
Creating the resources --> Create the resources
Without both of these values, your code does not work. --> You must have both of these values for your code to work.
you need to create --> you must create

Reply:
fixed